packages/config/src/events.js

```diff
@@ -0,0 +1,19 @@
+'use strict'
+
+// List of build plugins events
+const EVENTS = [
```

Comment:
In theory, this should belong in `@netlify/build`.
However, there are some times where we've been needing this inside `@netlify/config`. In this case, we validate that configuration mutations are not applied too late in the build process. This requires knowing both build events (`@netlify/build` responsibility) and configuration property names (`@netlify/config` responsibility). Therefore, it made sense here to move this array to `@netlify/config`.